docs/source/plugins.rst

```diff
@@ -38,7 +38,7 @@ for more information on RabbitMQ and how to consume the messages.
            self.channel = self.connection.channel()
            self.channel.queue_declare(queue='dask_task_status', durable=True)
 
-       def transition(self, key, start, finish, *args, **kwargs):
+       def transition(self, key, start, finish, *args, stimulus_id, **kwargs):
```

Comment:
This is another great example why we should just require/encourage kwargs. I don't think this should be mandatory